backend/CertificateManagerAPIs/Services/ServiceExtension.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.EntityFrameworkCore;
+namespace CertificateManagerAPIs.Services
+{
+    public static class ServiceExtension
```

Comment:
This is not service, this is extension method for connection string configuration, so should not be in Services folder

Reply:
This is also solved.
Thank you.